src/expr/src/scalar/func/impls.rs

```diff
@@ -8,27 +8,41 @@
 // by the Apache License, Version 2.0.
 
 mod boolean;
+mod byte;
 mod char;
+mod date;
 mod datum;
 mod float32;
 mod float64;
 mod int16;
 mod int32;
 mod int64;
+mod interval;
 mod numeric;
 mod oid;
 mod regproc;
 mod string;
+mod time;
+mod timestamp;
+mod uuid;
+mod varchar;
 
 pub use self::char::*;
+pub use self::uuid::*;
```

Comment:
NIT: alphabetical order. I only used the `self::` prefix for char because otherwise rust thinks I'm referring to the builtin `char`type.

Reply:
`self` is necessary because of the `uuid` crate, and prefixing with `self` autosorts it to this position.